libbeat/common/schema/mapstriface/mapstriface.go

```diff
@@ -171,6 +171,11 @@ func Bool(key string, opts ...schema.SchemaOption) schema.Conv {
 	return schema.SetOptions(schema.Conv{Key: key, Func: toBool}, opts)
 }
 
+// Float creates a Conv object for parsing floats
+func Float(key string, opts ...schema.SchemaOption) schema.Conv {
+	return schema.SetOptions(schema.Conv{Key: key, Func: toInteger}, opts)
```

Comment:
You use the `toInteger` function here which I think does not return the expected result?